packages/main/src/modules/bin.ts

```diff
@@ -287,30 +294,34 @@ export function run(pkg: string, bin: string, options: RunOptions = {}): Child {
     }
   });
 
-  function handleStream(stream: NodeJS.ReadableStream) {
-    stream!.on('data', (data: Buffer) => handleData(data, matchers));
+  function handleStream(stream: NodeJS.ReadableStream, type: StreamType) {
+    stream!.on('data', (data: Buffer) => handleData(data, matchers, type));
   }
 
-  handleStream(forked.stdout!);
-  handleStream(forked.stderr!);
+  handleStream(forked.stdout!, 'stdout');
+  handleStream(forked.stderr!, 'stderr');
```

Comment:
I think we can remove this since it's duplicated with the event handlers at the top. 
Also I think that having multiple `data` events for the same stream doesn't work properly since the data will be handled by one of them and the others will receive an empty flow of data. (not sure about the last one tho)